.github/workflows/firebase-hosting-preview.yml

```diff
@@ -41,21 +41,82 @@ jobs:
         with:
           node-version: 'lts/*'
 
-      - name: Install dependencies and build
-        run: |
-          npm ci
-          npm run build
+      - name: Install dependencies
+        run: npm ci
+
+      # Inject the App Check Debug Token into the .env file at build time
+      - name: Create .env file with App Check Debug Token
+        run: echo "VITE_APPCHECK_DEBUG_TOKEN=${{ secrets.VITE_APPCHECK_DEBUG_TOKEN }}" >> .env
```

Comment:
Is this part still necessary now that we are adding the preview url to the whitelist?